compiler-plugin-tests/src/test/resources/static_code_analyzer/ballerina_packages/rule1/main.bal

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2024 WSO2 LLC. (https://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/log;
+
+public function main() {
+    log:printInfo(password);
+    log:printError(string `Error: ${password}`);
+    log:printError("Error " + password);
+    log:printWarn("Warning", password = password);
+    log:printError("Error", password = password, user = user);
```

Comment:
Shall we add a test with sensitive variable and configurable at same time.
```
log:printWarn(password, configVar);
```

Reply:
Fixed in [3edb530](https://github.com/ballerina-platform/module-ballerina-log/pull/1175/commits/3edb5309e84fdca1986162244a181483a3bf3b2e)